initrdsrc/init

```diff
@@ -42,6 +42,9 @@ add_layers() {
 }
 
 add_layers rootfs
+if [ -f /boot/ssh_authorized_keys ]; then
+    add_layers sshd
```

Comment:
Does the layer shadow any common files like passwd?  It probably won't break anything, but a later layer may break it.

Does rootfs have extra passwd entries?

Reply:
it does not, and I only know that because by inspecting the generated `sshd.off` by hand.. but it's probably worth adding a test to make sure we know when that happens in the future.

Reply:
That would be nice, though this feature will only be used for debugging.

The layers as directory trees are in build/<arch>/fs/.  They are named by docker ids, so look in build/imgfs/layers/*.layers for the layer ids that make up each container.

Reply:
added a test in ee6cd07 that can hopefully easily generalize in case we ever do this for non-debug stuff in the future. PTAL @squeakbat?